shared/.storybook/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const webpack = require('webpack')
+// you can use this file to add your custom webpack plugins, loaders and anything you like.
+// This is just the basic way to add additional webpack configurations.
+// For more information refer the docs: https://storybook.js.org/configurations/custom-webpack-config
+
+const fileLoaderRule = {
+  loader: 'file-loader',
+  options: {
+    name: '[name].[ext]',
+  },
+}
+
+module.exports = (storybookBaseConfig, configType) => {
```

Comment:
i'm trying to pull as little webpack config from ours as possible. hence the copy and pasting. my hope is that this won't need to grow